src/game/ui/textbtn.cpp

```diff
@@ -124,30 +100,26 @@ void TextButtonClass::Set_Text(int str_id, BOOL adjust)
     }
 }
 
-void TextButtonClass::Set_Style(TextPrintType style)
-{
-    TextStyle = style;
-}
-
 void TextButtonClass::Draw_Background()
 {
     BoxStyleEnum style;
 
     if (HasOutline) {
         g_logicPage->Draw_Rect(XPos - 1, YPos - 1, Width + XPos + 2, Height + YPos + 2, COLOR_BLACK);
+    }
 
-        // Draw an drop shadow underneath the button if required.
```

Comment:
The style change for the comments is somewhat pointless, I'd recommend just dropping the commented out code all together unless/until we want to actually implement it as its in the commit history if we want to pull it back.